common_logging/src/main/java/uk/gov/dhsc/htbhf/logging/event/EventFailedEvent.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.dhsc.htbhf.logging.event;
+
+import lombok.Builder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents an event that has failed.
+ */
+public class EventFailedEvent extends Event {
+
+    public static final String FAILED_EVENT_KEY = "failedEvent";
+    public static final String FAILURE_DESCRIPTION_KEY = "failureDescription";
+
+    @Builder
+    public EventFailedEvent(String failureDescription, Event failedEvent) {
```

Comment:
It took me a while to figure out why this was taking in an `Event` on construction so it might be worthwhile having a bit of JavaDoc explaining its usage.

Reply:
I'd also do a check here to make sure the `failedEvent` isn't null, I can foresee the possibility in a failure scenario where the original event might be null.

Reply:
hopefully javadoc is no longer necessary/